plugin/core/sessions.py

```diff
@@ -1091,8 +1092,8 @@ def _handle_initialize_success(self, result: Any) -> None:
             pattern = config.get('pattern')
             if pattern:
                 events = config.get('events') or ['create', 'change', 'delete']
-                ignores = config.get('ignores') or DEFAULT_IGNORES
                 for folder in self.get_workspace_folders():
+                    ignores = config.get('ignores') or self._get_global_ignore_globs(folder.path)
```

Comment:
I'm not sure how to exactly handle the situation when the `ignores` is specified in the config. Whether all of those should be combined or not. If we combine then there will be no way to opt-out of the excluded patterns.

Reply:
I would leave as is. (I would just add a comment above with the explanation why they are not combined)